src/cosim/fmi/v1/fmu.cpp

```diff
@@ -166,9 +166,11 @@ namespace
 {
 void step_finished_placeholder(fmi1_component_t, fmi1_status_t)
 {
+#ifndef LIBCOSIM_NO_FMI_LOGGING
```

Comment:
`LIBCOSIM_NO_FMI_LOGGING` was introduced to supress logging comming from the FMUs itself, but this supresses libcosim generated output. I'm not sure that is a good idea.

Reply:
I think this function is assigned to the fmi callback instance (`fmi(1|2)_callback_functions_t`), which is called by the fmu?

Reply:
It is still not connected to the logging part. So IMO, the macro is taking liberties here. However, I think it is safe to remove the current logic altogether. We are not using, nor planning to use this feature.

Reply:
Shouldn't we define the callback as it can be called by FMUs? We can of course remove the logging and leave a comment describing it's existence.

Reply:
stepFinished callback can be NULL per the standard.

Reply:
Actually it SHOULD be NULL:
`Optional call back function to signal if the computation of a communication step of a cosimulation slave is finished. A null pointer can be provided. In this case the master must
use fmiGetStatus(..) to query the status of fmi2DoStep. If a pointer to a function
is provided, it must be called by the FMU after a completed communication step.`

Reply:
Good catch! For references this is stated in section 2.1.5 (p 21) in the FMI 2.0.1.
Let's use a null pointer.

Reply:
Updated